src/java/org/apache/cassandra/utils/ReflectionUtils.java

```diff
@@ -105,4 +105,36 @@ public static <K, V> void clearMapField(Class<?> clazz, Object instance, String
             throw new RuntimeException(String.format("Could not clear map field %s in class %s", mapName, clazz), ex);
         }
     }
+
+    public static void setFieldToNull(Object object, String fieldName)
```

Comment:
I think it would be better to accept the declaring class here, so we know we're nulling the correct field while also simplifying the implementation.